inst/templates/ad_advs.R

```diff
@@ -47,13 +47,8 @@ bmi_for_age <- who_bmi_for_age_boys %>%
       AGE = round(AGE * 30.4375)
     ) %>%
     # Interpolate the AGE by SEX
-    group_by(SEX) %>%
     # Ensure first that Age unit is "DAYS"
-    do(derive_interp_records(., parameter = "BMI")) %>%
-    ungroup() %>%
-    # Keep patients >= 2 yrs only - Remove duplicates for 730 Days old which
-    # must come from WHO metadata only
-    filter(AGE >= 730.5)) %>%
+    derive_interp_records(., parameter = "BMI")) %>%
```

Comment:
See comments above about `by_vars` and update template in similar way. also i think styler will probably want us to write it more like: 

```
derive_interp_records(by_vars = exprs(SEX),
                      parameter = "HEIGHT")
```

I think through using `%>%` the `.,` can be avoided but tell me if i'm wrong.

Reply:
implemented